mkdocs_include_markdown_plugin/event.py

```diff
@@ -586,20 +595,54 @@ def found_include_markdown_tag(match):
         )
 
     markdown = re.sub(
-        INCLUDE_TAG_REGEX,
+        include_tag_regex,
         found_include_tag,
         markdown,
     )
     return re.sub(
-        INCLUDE_MARKDOWN_TAG_REGEX,
+        include_markdown_tag_regex,
         found_include_markdown_tag,
         markdown,
     )
 
 
-def on_page_markdown(markdown, page, docs_dir):
+def on_page_markdown(
+    markdown,
+    page,
+    docs_dir,
+    opening_tag='{%',
+    closing_tag='%}',
+):
+
+    # Escape special characters of the tags with a backslash.
+    # For example : '{%' will be escaped as '\{\%'
+    def escape(text):
+        TO_BE_ESCAPED = '.^$*+-?{}[]\\|():<>=!/#%,;'
+        return ''.join([('\\' if i in TO_BE_ESCAPED else '')+i for i in text])
+    escaped_opening_tag = escape(opening_tag)
+    escaped_closing_tag = escape(closing_tag)
```

Comment:
I think that you're reinveting the wheel here, you can use [`re.escape`](https://docs.python.org/3/library/re.html#re.escape):

```suggestion
    escaped_opening_tag = re.escape(opening_tag)
    escaped_closing_tag = re.escape(closing_tag)
```